src/components/DialogBubble.tsx

```diff
@@ -17,9 +11,12 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const mui = useTheme();
   const styles = useStyles(mui);
+  const white = mui.palette.common.white;
   return (
-    <Stack sx={styles.bubbleContainer} aria-haspopup="true">
-      <Box sx={styles.bubblePointer} />
+    <Stack
+      sx={{ ...styles.bubbleContainer, bgcolor: white }}
```

Comment:
```suggestion
      sx={{ ...styles.bubbleContainer, bgcolor: mui.palette.common.white }}
```
you are using it only here